docs/user_guide/assessment.rst

```diff
@@ -468,6 +470,21 @@ argument. Here is an example of a pie chart.
 There are many other customizations that can be done. More information can be found in
 :meth:`pandas.DataFrame.plot`.
 
+
+.. _objectSavingLoading:
+
+Object Saving & Loading
+-----------------------
+
+Metrics can be saved to and loaded from files using the 'pickle <https://docs.python.org/3/library/pickle.html>' package.
+
+.. doctest:: assessment_metrics
+
+	>>> mf = MetricFrame(...)
+	>>> pickle.dump(mf, open(file_name, 'wb'))
+	>>> pickle.load(open(file_name, 'rb'))
```

Comment:
Wonderful! We should probably show that the loaded object has the contents by printing its by_group property.